app/models/ability.rb

```diff
@@ -61,6 +63,10 @@ def can_crud_team(user_id)
       can :destroy, Team, owners: { id: user_id }
     end
 
+    def can_assign_membership_by_email(user)
```

Comment:
@bakoe I had to rename the ability to match the controller action (otherwise I received a `permission denied` when sending a POST request to `can_assign_membership`). I'm open to discuss which is the best way of naming the action/ability (we could also omit the ...by_email suffix).

Reply:
Following the YAGNI principle I agree with having only the (quite specific) `(can_)assign_membership_by_email` ability and controller method right now. Whenever a more generic `(can_)assign_membership` ability and/or controller method are needed, one can introduce them by factoring out common behavior. Because at the moment only membership assignment via email is requested, I would leave the naming with `_by_email` suffix.